.github/workflows/release-please.yml

```
@@ -8,13 +8,15 @@ name: release-please
 jobs:
   release-please:
     runs-on: ubuntu-latest
+    permissions: write-all
     outputs:
       release_created: ${{ steps.release.outputs.release_created }}
     steps:
       - uses: actions/checkout@v4
       - uses: GoogleCloudPlatform/release-please-action@v4
         id: release
         with:
+          token: ${{ secrects.PAT}}
```

actionlint (1.7.4)

undefined variable "secrects". available variables are "env", "github", "inputs", "job", "matrix", "needs", "runner", "secrets", "steps", "strategy", "vars"

(expression)